openshift/templates/pims-api/deploy.yaml

```diff
@@ -18,34 +20,65 @@ objects:
   - kind: Route
     apiVersion: v1
     metadata:
-      name: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}
+      name: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}${ID}
+      namespace: ${PROJECT_NAMESPACE}-${ENV_NAME}
       labels:
-        name: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}
+        name: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}${ID}
         app: ${APP_NAME}
         component: ${COMP_NAME}
         env: ${ENV_NAME}
         role: backend
+        id: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}${ID}
     spec:
-      host: ${APPLICATION_DOMAIN}
-      path: ${DOMAIN_PATH}
+      host: ${APP_DOMAIN}
+      path: ${API_PATH}
       to:
         kind: Service
-        name: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}
+        name: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}${ID}
         weight: 100
       port:
         targetPort: ${APP_PORT}-tcp
+      tls:
+        insecureEdgeTerminationPolicy: Redirect
+        termination: edge
+  - kind: Route
+    apiVersion: v1
+    metadata:
+      name: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}${ID}-docs
+      namespace: ${PROJECT_NAMESPACE}-${ENV_NAME}
+      labels:
+        name: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}${ID}-docs
+        app: ${APP_NAME}
+        component: ${COMP_NAME}
+        env: ${ENV_NAME}x
+        role: backend
+        id: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}${ID}
+    spec:
+      host: ${APP_DOMAIN}
+      path: ${API_DOCS_PATH}
+      to:
+        kind: Service
+        name: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}${ID}
```

Comment:
should this be -docs as well?

Reply:
No, this is a reference to the Service name.